### ISSUE TITLE
swagger plugin generates code that throws Cannot find module '../../../stream'

### ISSUE THREAD

**Issue by tom-thorne** [nestjs/swagger#1106]
## I'm submitting a...

<pre><code>
[ ] Regression 
[ x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

When a route returns a type that is defined in a core nodejs module, the distribution js that is created has an invalid require path.

For example

```
import { Readable } from 'stream';
...
    @Get(':id')
    async getExport(@Param('id') id: string): Promise<Readable> {
        return getExportFileStream(id);
    }   
```
generates:

```
__decorate([
    common_1.Get(':id'),
    openapi.ApiResponse({ status: 200, type: require("../../../stream").Readable }),  // <-- this is invalid
  ....
], EnrichmentImportExportController.prototype, "getExport", null);
```

## Expected behavior

It creates a valid path

```
__decorate([
    common_1.Get(':id'),
    openapi.ApiResponse({ status: 200, type: require("stream").Readable }), 
  ....
], EnrichmentImportExportController.prototype, "getExport", null);
```

## Environment

<pre><code>
Nest version: 7.5.1
"@nestjs/swagger": "4.7.6"
</code></pre>

**Comment by kamilmysliwiec** [comment]
Please provide a minimum reproduction repository.

**Comment by AlessandroMrli** [comment]
I had a similar issue
I needed to override the default by using this
	@ApiResponse({
		status: 200,
		type: Readable,
	})

**Comment by kamilmysliwiec** [comment]
Fixed in the latest release

**Comment by deviantfero** [comment]
I'm having this same issue with other libraries:
```
internal/modules/cjs/loader.js:797                                                                                                  
    throw err;                                                                                                                      
    ^                                                                                                                               
                                                                                                                                    
Error: Cannot find module '../../amazon-cognito-identity-js'                                                                        
```

```
__decorate([
    common_1.Post('refresh'),
    swagger_1.ApiExcludeEndpoint(),
    openapi.ApiResponse({ status: 201, type: require("../../amazon-cognito-identity-js").CognitoUserSession }),
    __param(0, common_1.Body()),
    __metadata("design:type", Function),
    __metadata("design:paramtypes", [refresh_token_dto_1.RefreshTokenDto]),
    __metadata("design:returntype", Promise)
], AuthController.prototype, "refreshToken", null);
```
the fix that is in the `4.8.0` release just adds an exception for the `stream` package, this will not work for types that are derived from a node package. I will try to submit a PR that fixes this
